custom_components/tahoma/tahoma_entity.py

```diff
@@ -111,7 +111,10 @@ def device_info(self) -> Dict[str, Any]:
             }
 
         manufacturer = self.select_state(CORE_MANUFACTURER_NAME_STATE) or "Somfy"
-        model = self.select_state(CORE_MODEL_STATE) or self.device.widget
+        model = (
+            self.select_state(CORE_MODEL_STATE, "core:ProductModelNameState")
```

Comment:
Missing the constant. This state  come from Hitachi devices?

Reply:
If I end up using this, I will make it a constant. However, I am still a bit in doubt here. It is a constant used by Hitachi devices, however it is changeable by the user...